src/cfnlint/rules/resources/ectwo/Ebs.py

```diff
@@ -26,13 +26,15 @@ def _checkEbs(self, cfn, ebs, path):
                 for volume_type_obj in volume_types_obj:
                     volume_type = volume_type_obj.get('Value')
                     if isinstance(volume_type, six.string_types):
-                        if volume_type == 'io1':
+                        if volume_type in ('io1', 'io2'):
```

Comment:
I wonder if this is a safe assumption going forward:
```suggestion
                        if volume_type.startswith('io'):
```

Reply:
I did consider that approach, but steered away from it.  When io3 comes out, will the Iops argument be required for it?  Maybe.  We don't know the answer.  What if AWS creates a different volume type that just happens to start with the letters "io"?

I don't feel good about letting the linter raise an error based on a "maybe".  In my view, we should only be telling the user their code is wrong, if we *know*, for sure, it is in fact wrong.

Reply:
agreed